event/module.modulemap

```diff
@@ -0,0 +1,6 @@
+module "event" {
```

Comment:
Why is this file here at all? What is this doing?

Reply:
I actually have no idea how this file was created